lightning/src/ln/chan_utils.rs

```diff
@@ -268,36 +268,41 @@ pub fn derive_public_revocation_key<T: secp256k1::Verification>(secp_ctx: &Secp2
 		Sha256::from_engine(sha).into_inner()
 	};
 
-	let mut part_a = revocation_base_point.clone();
-	part_a.mul_assign(&secp_ctx, &rev_append_commit_hash_key)?;
-	let mut part_b = per_commitment_point.clone();
-	part_b.mul_assign(&secp_ctx, &commit_append_rev_hash_key)?;
-	part_a.combine(&part_b)
+	let mut counterparty_contrib = revocation_base_point.clone();
+	counterparty_contrib.mul_assign(&secp_ctx, &rev_append_commit_hash_key)?;
+	let mut our_contrib = per_commitment_point.clone();
+	our_contrib.mul_assign(&secp_ctx, &commit_append_rev_hash_key)?;
+	counterparty_contrib.combine(&our_contrib)
 }
 
 /// The set of public keys which are used in the creation of one commitment transaction.
 /// These are derived from the channel base keys and per-commitment data.
 ///
+/// A broadcaster key is coming from potential broadcaster of the computed transaction.
+/// A countersignatory key is coming from a protocol participant unable to broadcast by its
+/// own the computed transaction.
+///
 /// These keys are assumed to be good, either because the code derived them from
 /// channel basepoints via the new function, or they were obtained via
 /// PreCalculatedTxCreationKeys.trust_key_derivation because we trusted the source of the
 /// pre-calculated keys.
 #[derive(PartialEq, Clone)]
 pub struct TxCreationKeys {
-	/// The per-commitment public key which was used to derive the other keys.
+	/// The broadcaster's per-commitment public key which was used to derive the other keys.
 	pub per_commitment_point: PublicKey,
-	/// The revocation key which is used to allow the owner of the commitment transaction to
-	/// provide their counterparty the ability to punish them if they broadcast an old state.
+	/// The broadcaster's revocation key which is used to allow the broadcaster of the commitment
+	/// transaction to provide their counterparty the ability to punish them if they broadcast
+	/// an old state.
 	pub revocation_key: PublicKey,
-	/// A's HTLC Key
-	pub a_htlc_key: PublicKey,
-	/// B's HTLC Key
-	pub b_htlc_key: PublicKey,
-	/// A's Payment Key (which isn't allowed to be spent from for some delay)
-	pub a_delayed_payment_key: PublicKey,
+	/// Broadcaster's HTLC Key
+	pub broadcaster_htlc_key: PublicKey,
+	/// Countersignatory's HTLC Key
+	pub countersignatory_htlc_key: PublicKey,
+	/// Broadcaster's Payment Key (which isn't allowed to be spent from for some delay)
+	pub delayed_payment_key: PublicKey,
```

Comment:
Should we leave the broadcaster_ prefix here?

Reply:
There is only one delayed_payment_key per-commitment transaction. Previously `revocation_key` didn't have a prefix. This is converging towards this nomenclature antecedent.

Reply:
Sure, its the only one, but there's no harm in indicating who's it is, and it can only help readability.